bittensor/_wallet/wallet_impl.py

```diff
@@ -87,6 +87,9 @@ def neuron(self, netuid: int) -> SimpleNamespace:
     def trust(self, netuid: int) -> SimpleNamespace:
         return self.get_neuron(netuid = netuid).trust
 
+    def validator_trust(self, netuid: int) -> SimpleNamespace:
+        return self.get_neuron(netuid = netuid).validator_trust
+
     def rank(self, netuid: int) -> SimpleNamespace:
         return self.get_neuron(netuid = netuid).rank
 
```

Comment:
The type hint for both should be `Optional[float]` or `float`. And possibly add a raises in a docstring if there's no None-neuron check